.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+---
+workspace_root: &workspace_root
+                  /go/src/github.com/hashicorp/terraform-aws-nomad
+
+defaults: &defaults
+  working_directory: *workspace_root
+  docker:
+    - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:latest
+
+version: 2
+jobs:
+  validate_terraform:
+    docker:
+      - image: hashicorp/terraform
+    steps:
+      - checkout
+      - run:
+          name: Validate Terraform Formatting
+          command: "[ -z \"$(terraform fmt -write=false)\" ] || { terraform fmt -write=false -diff; exit 1; }"
```

Comment:
Interesting approach! We also have [pre-commit hooks](https://github.com/gruntwork-io/pre-commit) for some of this stuff.

Reply:
Ah ok, I sought inspiration from the `terraform-aws-consul` -repo and though this was the way to go with the Hashicorp -modules.